flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/model/FormBuilder.java

```diff
@@ -214,6 +214,22 @@ public FormBuilder<T> returnType(String value, String typeConstraint, boolean op
         return this;
     }
 
+    public FormBuilder<T> description(String value, String typeConstraint, boolean optional) {
```

Comment:
Are there cases where the description is mandatory? If it is always optional, better if we can remove this parameter as well